### PR TITLE
View and networking

### PR DESCRIPTION
**Pull request by AldrinBahle**
Hi Please review the work I've accomplished thus far.

### CODE REVIEW

File: Mars Mission/Mars Mission/Model/WeatherModel.swift [hunk 1]
```diff
@@ -8,28 +8,16 @@
...
-        let temperatureHigh: Double?
-        let icon: String?
-        let humidity: Double?
-    }
+    var forecasts: [WeatherAtrr]!
+    var lastUpdated: String?
+    var weatherStation: String?
 }
+
+struct WeatherAtrr: Codable {
```
Comment (Sashen943) [inline Mars Mission/Mars Mission/Model/WeatherModel.swift]: This class should be name Forecast

File: Mars Mission/Mars Mission/ViewModel/WeatherViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    self.handleThatFetchPostFailure(error)
+                }
+            }
+        }
+    }
+    
+    private func handleThatFetchPostSucceeds(_ post: WeatherDataModel) {
+        DispatchQueue.main.async {
+            self.post = post
+            self.view.populateWeather(post.forecasts[0].date ?? "",
```
Comment (Sashen943) [inline Mars Mission/Mars Mission/ViewModel/WeatherViewModel.swift]: Please fix looping here

**Review by Sashen943 — CHANGES_REQUESTED** [review]

**Comment by AldrinBahle** [comment]
Screenshot 2020-11-09 at 13 53 25


All the changes requested are now attended.

**Review by Sashen943 — APPROVED** [review]
Some code standard issues like variable naming and commented out code but happy to approve

**Comment by AldrinBahle** [comment]
Thank you Sir.

**Comment by AldrinBahle** [comment]
Thank you Sir all comments noted.

**Pull request merged**